File: hotel_search/v2/get/hotel_offers_by_hotel/iOS SDK/HotelSearch.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// Get list of offers for a specific hotel
amadeus.shopping.hotelOfferByHotel.get(params: ["hotelId": "BGMILBGB",
                                                "adults": "2",
                                                "roomQuantity": "1",
                                                "paymentPolicy": "NONE"],
    onCompletion: { result in
        switch result {
        case let .success(response):
            print(response.data)
        case let .failure(error):
            fatalError(error.localizedDescription)
        }
    })

File: hotel_search/v3/get/hotel_offers_offerId/Node SDK/hotel_search.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Check offer conditions of a specific offer id
    const response = await amadeus.shopping
      .hotelOfferSearch(
        "63A93695B58821ABB0EC2B33FE9FAB24D72BF34B1BD7D707293763D8D9378FC3"
      )
      .get();

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
